### ISSUE TITLE
关于准确率

### ISSUE THREAD

**Issue by ynu-yangpeng** [yuanpengtu/SANM#7]
你好，我对你们的工作非常感兴趣，我觉得你们的想法非常有意思，我在复现你们论文的过程中出现了一下问题，望能得到解答。
（1）我使用18-layer PreAct ResNet 按照batch_size=64，learning_rat=0.02，noise_mode=sym,noise ratio=0.8,lambda_u=25,num_epochs=300,p_threshold=0.5,seed=123,训练得到的结果如下：
Epoch:0   Accuracy:2.86
Epoch:1   Accuracy:3.30
Epoch:2   Accuracy:5.41
Epoch:3   Accuracy:5.70
Epoch:4   Accuracy:7.09
Epoch:5   Accuracy:6.32
Epoch:6   Accuracy:6.60
Epoch:7   Accuracy:7.70
Epoch:8   Accuracy:8.76
Epoch:9   Accuracy:9.19
Epoch:10   Accuracy:9.48
Epoch:11   Accuracy:10.13
Epoch:12   Accuracy:10.45
Epoch:13   Accuracy:10.07
Epoch:14   Accuracy:11.35
Epoch:15   Accuracy:12.31
Epoch:16   Accuracy:12.44
Epoch:17   Accuracy:12.59
Epoch:18   Accuracy:13.69
Epoch:19   Accuracy:14.18
Epoch:20   Accuracy:13.34
Epoch:21   Accuracy:16.37
Epoch:22   Accuracy:18.10
Epoch:23   Accuracy:19.79
Epoch:24   Accuracy:21.35
Epoch:25   Accuracy:22.32
Epoch:26   Accuracy:23.84
Epoch:27   Accuracy:24.75
Epoch:28   Accuracy:25.74
Epoch:29   Accuracy:27.20
Epoch:30   Accuracy:28.90
Epoch:31   Accuracy:29.98
Epoch:32   Accuracy:30.63
Epoch:33   Accuracy:31.55
Epoch:34   Accuracy:32.32
Epoch:35   Accuracy:33.03
Epoch:36   Accuracy:32.80
Epoch:37   Accuracy:33.93
Epoch:38   Accuracy:34.33
Epoch:39   Accuracy:34.59
Epoch:40   Accuracy:36.15
Epoch:41   Accuracy:36.71
Epoch:42   Accuracy:36.65
Epoch:43   Accuracy:37.32
Epoch:44   Accuracy:37.67
Epoch:45   Accuracy:38.22
Epoch:46   Accuracy:38.35
Epoch:47   Accuracy:37.80
Epoch:48   Accuracy:38.83
Epoch:49   Accuracy:38.95
Epoch:50   Accuracy:39.11
Epoch:51   Accuracy:39.91
Epoch:52   Accuracy:39.83
Epoch:53   Accuracy:40.58
Epoch:54   Accuracy:41.34
Epoch:55   Accuracy:40.66
Epoch:56   Accuracy:41.33
Epoch:57   Accuracy:40.85
Epoch:58   Accuracy:41.13
Epoch:59   Accuracy:41.73
Epoch:60   Accuracy:41.70
Epoch:61   Accuracy:42.52
Epoch:62   Accuracy:42.36
Epoch:63   Accuracy:42.45
Epoch:64   Accuracy:42.61
Epoch:65   Accuracy:43.03
Epoch:66   Accuracy:42.91
Epoch:67   Accuracy:43.55
Epoch:68   Accuracy:42.98
Epoch:69   Accuracy:43.15
Epoch:70   Accuracy:43.42
Epoch:71   Accuracy:43.98
Epoch:72   Accuracy:43.45
Epoch:73   Accuracy:43.22
Epoch:74   Accuracy:43.35
Epoch:75   Accuracy:44.55
Epoch:76   Accuracy:43.92
Epoch:77   Accuracy:44.75
Epoch:78   Accuracy:43.82
Epoch:79   Accuracy:44.09
Epoch:80   Accuracy:44.23
Epoch:81   Accuracy:43.79
Epoch:82   Accuracy:44.95
Epoch:83   Accuracy:44.93
Epoch:84   Accuracy:44.84
Epoch:85   Accuracy:45.05
Epoch:86   Accuracy:45.26
Epoch:87   Accuracy:44.98
Epoch:88   Accuracy:45.01
Epoch:89   Accuracy:44.98
Epoch:90   Accuracy:44.66
Epoch:91   Accuracy:44.62
Epoch:92   Accuracy:44.35
Epoch:93   Accuracy:45.12
Epoch:94   Accuracy:45.14
Epoch:95   Accuracy:45.30
Epoch:96   Accuracy:44.87
Epoch:97   Accuracy:45.31
Epoch:98   Accuracy:45.53
Epoch:99   Accuracy:45.65
Epoch:100   Accuracy:46.06
Epoch:101   Accuracy:45.24
Epoch:102   Accuracy:45.28
Epoch:103   Accuracy:45.97
Epoch:104   Accuracy:45.79
Epoch:105   Accuracy:46.14
Epoch:106   Accuracy:46.07
Epoch:107   Accuracy:46.04
Epoch:108   Accuracy:46.07
Epoch:109   Accuracy:45.89
Epoch:110   Accuracy:45.82
Epoch:111   Accuracy:45.02
Epoch:112   Accuracy:46.55
Epoch:113   Accuracy:45.73
Epoch:114   Accuracy:45.70
Epoch:115   Accuracy:46.32
Epoch:116   Accuracy:45.48
Epoch:117   Accuracy:45.79
Epoch:118   Accuracy:45.53
Epoch:119   Accuracy:45.97
Epoch:120   Accuracy:46.15
Epoch:121   Accuracy:45.72
Epoch:122   Accuracy:45.30
Epoch:123   Accuracy:46.68
Epoch:124   Accuracy:46.04
Epoch:125   Accuracy:45.67
Epoch:126   Accuracy:45.40
Epoch:127   Accuracy:46.26
Epoch:128   Accuracy:45.88
Epoch:129   Accuracy:45.27
Epoch:130   Accuracy:46.12
Epoch:131   Accuracy:46.51
Epoch:132   Accuracy:46.03
Epoch:133   Accuracy:46.68
Epoch:134   Accuracy:43.98
Epoch:135   Accuracy:45.81
Epoch:136   Accuracy:44.77
Epoch:137   Accuracy:45.16
Epoch:138   Accuracy:44.39
Epoch:139   Accuracy:46.70
Epoch:140   Accuracy:46.33
Epoch:141   Accuracy:47.07
Epoch:142   Accuracy:46.35
Epoch:143   Accuracy:46.43
Epoch:144   Accuracy:46.15
Epoch:145   Accuracy:47.98
Epoch:146   Accuracy:46.84
Epoch:147   Accuracy:44.35
Epoch:148   Accuracy:43.49
Epoch:149   Accuracy:46.67
Epoch:150   Accuracy:48.04
Epoch:151   Accuracy:48.01
Epoch:152   Accuracy:48.11
Epoch:153   Accuracy:48.15
Epoch:154   Accuracy:48.22
Epoch:155   Accuracy:48.66
Epoch:156   Accuracy:48.92
Epoch:157   Accuracy:48.70
Epoch:158   Accuracy:48.91
Epoch:159   Accuracy:48.83
Epoch:160   Accuracy:49.07
Epoch:161   Accuracy:48.90
Epoch:162   Accuracy:48.87
Epoch:163   Accuracy:48.74
Epoch:164   Accuracy:49.60
Epoch:165   Accuracy:48.55
Epoch:166   Accuracy:49.57
Epoch:167   Accuracy:48.44
Epoch:168   Accuracy:48.94
Epoch:169   Accuracy:49.18
Epoch:170   Accuracy:49.69
Epoch:171   Accuracy:48.89
Epoch:172   Accuracy:49.97
Epoch:173   Accuracy:48.79
Epoch:174   Accuracy:49.65
Epoch:175   Accuracy:48.90
Epoch:176   Accuracy:49.89
Epoch:177   Accuracy:48.75
Epoch:178   Accuracy:49.91
Epoch:179   Accuracy:48.77
Epoch:180   Accuracy:50.29
Epoch:181   Accuracy:49.48
Epoch:182   Accuracy:50.09
Epoch:183   Accuracy:50.69
Epoch:184   Accuracy:49.70
Epoch:185   Accuracy:49.73
Epoch:186   Accuracy:49.84
Epoch:187   Accuracy:49.36
Epoch:188   Accuracy:50.02
Epoch:189   Accuracy:49.28
Epoch:190   Accuracy:49.55
Epoch:191   Accuracy:49.49
Epoch:192   Accuracy:49.91
Epoch:193   Accuracy:48.79
Epoch:194   Accuracy:48.62
Epoch:195   Accuracy:48.61
Epoch:196   Accuracy:48.99
Epoch:197   Accuracy:49.70
Epoch:198   Accuracy:48.98
Epoch:199   Accuracy:48.78
Epoch:200   Accuracy:49.10
Epoch:201   Accuracy:50.47
Epoch:202   Accuracy:49.08
Epoch:203   Accuracy:48.58
Epoch:204   Accuracy:49.14
Epoch:205   Accuracy:49.07
Epoch:206   Accuracy:48.63
Epoch:207   Accuracy:48.46
Epoch:208   Accuracy:48.65
Epoch:209   Accuracy:49.21
Epoch:210   Accuracy:48.80
Epoch:211   Accuracy:48.39
Epoch:212   Accuracy:48.48
Epoch:213   Accuracy:48.55
Epoch:214   Accuracy:49.00
Epoch:215   Accuracy:49.17
Epoch:216   Accuracy:48.63
Epoch:217   Accuracy:49.06
Epoch:218   Accuracy:48.12
Epoch:219   Accuracy:47.55
Epoch:220   Accuracy:47.44
Epoch:221   Accuracy:47.90
Epoch:222   Accuracy:47.94
Epoch:223   Accuracy:51.78
Epoch:224   Accuracy:52.15
Epoch:225   Accuracy:52.41
Epoch:226   Accuracy:52.27
Epoch:227   Accuracy:52.52
Epoch:228   Accuracy:52.56
Epoch:229   Accuracy:52.60
Epoch:230   Accuracy:52.91
Epoch:231   Accuracy:52.43
Epoch:232   Accuracy:52.03
Epoch:233   Accuracy:52.42
Epoch:234   Accuracy:52.64
Epoch:235   Accuracy:52.37
Epoch:236   Accuracy:52.79
Epoch:237   Accuracy:52.58
Epoch:238   Accuracy:52.46
Epoch:239   Accuracy:52.63
Epoch:240   Accuracy:52.31
Epoch:241   Accuracy:52.36
Epoch:242   Accuracy:52.67
Epoch:243   Accuracy:52.73
Epoch:244   Accuracy:52.61
Epoch:245   Accuracy:52.71
Epoch:246   Accuracy:52.65
Epoch:247   Accuracy:52.81
Epoch:248   Accuracy:53.22
Epoch:249   Accuracy:52.69
Epoch:250   Accuracy:52.77
Epoch:251   Accuracy:52.85
Epoch:252   Accuracy:52.63
Epoch:253   Accuracy:53.00
Epoch:254   Accuracy:52.88
Epoch:255   Accuracy:52.98
Epoch:256   Accuracy:52.87
Epoch:257   Accuracy:52.88
Epoch:258   Accuracy:52.51
Epoch:259   Accuracy:52.69
Epoch:260   Accuracy:52.97
Epoch:261   Accuracy:53.05
Epoch:262   Accuracy:53.04
Epoch:263   Accuracy:52.48
Epoch:264   Accuracy:52.92
Epoch:265   Accuracy:52.78
Epoch:266   Accuracy:52.64
Epoch:267   Accuracy:53.62
Epoch:268   Accuracy:53.76
Epoch:269   Accuracy:52.94
Epoch:270   Accuracy:53.15
Epoch:271   Accuracy:52.73
Epoch:272   Accuracy:52.92
Epoch:273   Accuracy:53.15
Epoch:274   Accuracy:52.91
Epoch:275   Accuracy:52.69
Epoch:276   Accuracy:53.06
Epoch:277   Accuracy:53.51
Epoch:278   Accuracy:52.69
Epoch:279   Accuracy:53.12
Epoch:280   Accuracy:52.96
Epoch:281   Accuracy:53.22
Epoch:282   Accuracy:52.30
Epoch:283   Accuracy:53.46
Epoch:284   Accuracy:53.26
Epoch:285   Accuracy:52.65
Epoch:286   Accuracy:53.14
Epoch:287   Accuracy:52.76
Epoch:288   Accuracy:53.30
Epoch:289   Accuracy:52.93
Epoch:290   Accuracy:53.06
Epoch:291   Accuracy:52.58
Epoch:292   Accuracy:52.61
Epoch:293   Accuracy:52.96
Epoch:294   Accuracy:53.40
Epoch:295   Accuracy:53.59
Epoch:296   Accuracy:53.20
Epoch:297   Accuracy:52.97
Epoch:298   Accuracy:52.12
Epoch:299   Accuracy:52.59
Epoch:300   Accuracy:53.05
（2）我使用原生resnet18的网络，加载自监督对比训练模型准确率一直在1%-2%，实验配置和上面的相同。

如果可以的话望你能指出我训练时哪里没有弄对，十分感谢！
